tensorflow/core/kernels/data/BUILD

```diff
@@ -193,21 +193,12 @@ tf_cc_test(
     ],
 )
 
-cc_library(
```

Comment:
this looks wrong ... note that `window_dataset` and `window_dataset_op` are two different targets and they are both used ... the former is not an op (but it is used by a couple of ops)

Reply:
Yeah, that's the part I'm not very sure. 

I just want to clarify that the code inside `window_dataset` is still kept. They (the function `NewWindowDataset()` and the class `WindowDataset : public DatasetBase`) are moved to `window_dataset_op` ([here](https://github.com/tensorflow/tensorflow/pull/29901/commits/417d08238a9730767e42ffdaa59d06f5e4b44798#diff-16c5e47751e6f14d041c887e72bfaa6fR366) and [here](https://github.com/tensorflow/tensorflow/pull/29901/commits/417d08238a9730767e42ffdaa59d06f5e4b44798#diff-16c5e47751e6f14d041c887e72bfaa6fR486)). The reason that I put them together is that `window_dataset_op` uses `WindowDataset` internally by calling the function `NewWindowDataset()`. By searching the code, I found `GroupByWindowDatasetOp` is using `WindowDataset` and have updated the include there.

You have a better understanding of this than me. I will be glad to change them back if you think it is better to keep them in separated files.

Reply:
Sounds good. Will move it to dataset_utils.

Reply:
After checking `dataset_utils`, it seems better to keep them in `window_dataset.h` and `window_dataset.cc` (keep them as same as they are currently). Please feel free to let me know if you want to move them anywhere.